sprints/sprint_1.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>OSCD</title>
-    <link rel="stylesheet" href="../hack.css">
+    <link rel="stylesheet" href="../hack_black.css">
```

Comment:
let's left sprint stiles as they are %) better create a new stile for the new sprint and not apply it for the others

Reply:
New style for every sprint is a good idea, but I'd still change the default one to `hack_clack.css`.

Reply:
then the main page will have a style of the second sprint %) let the main page be simple, neutral, and clean

Reply:
Guess what, I visited the site again and today I agree (: